alinka/widget/containers/main_body/content/__init__.py

```diff
@@ -16,10 +16,33 @@ def __init__(self, parent: QWidget):
         self.settings_container = SettingsContainer(self, visible=True)
         layout.addWidget(self.settings_container)
 
+    def validate_basic_settings(self) -> bool:
+        header_container = self.main_body_container.header_container
+        sidebar_menu_container = self.main_body_container.central_widget.side_bar.sidebar_menu_container
+        if not self.settings_container.is_valid:
+            error_message = self.settings_container.error_message
+            header_container.set_error_message(error_message)
+            sidebar_menu_container.toggle_create_application_btn(False)
+            self.show_settings_container()
+            return False
+        else:
+            header_container.clear_error_message()
+            sidebar_menu_container.toggle_create_application_btn(True)
+            return True
+
+    def showEvent(self, event):
```

Comment:
camelCase

Any reason you’re typing just some of the methods?

Reply:
https://doc.qt.io/qtforpython-6/PySide6/QtWidgets/QWidget.html#PySide6.QtWidgets.QWidget.showEvent

Reply:
I forgot they do that. :)

Reply:
For future reference, there is an option to switch to `snake_case` in PyQT 6: https://www.qt.io/blog/qt-for-python-6-released 🐍

Reply:
I'd love to this. But after second thought, it might be convenient differentiation  between required Qt method, and those added by us.